tests/vxlan/__init__.py

```diff
@@ -14,27 +17,35 @@ def pytest_addoption(parser):
     parser.addoption(
         "--num_vnet",
         action="store",
-        default=None,
+        default=8,
         type=int,
         help="number of VNETs for VNET VxLAN test"
     )
 
     parser.addoption(
         "--num_routes",
         action="store",
-        default=None,
+        default=16000,
         type=int,
         help="number of routes for VNET VxLAN test"
     )
 
     parser.addoption(
         "--num_endpoints",
         action="store",
-        default=None,
+        default=4000,
```

Comment:
Why is this default values changed?

Reply:
The parameters `num_vnet` and `num_intf_per_vnet` required a pytest default parameter to be able to print a warning if the number of RIFs exceeded 128. I added defaults for the other parameters (which match the default values given in the `vnet_config.j2` template) for consistency.

Reply:
The warning is [here](https://github.com/Azure/sonic-mgmt/pull/2236/files#diff-8399274c72942792acb3c281a7db7c9bR140) and the default values in the J2 template are defined around [here](https://github.com/Azure/sonic-mgmt/blob/d190bb2a5582ec3fda3aa2644c87e84f0c0fbab4/tests/vxlan/templates/vnet_config.j2#L41)